src/content/en/updates/2019/07/chrome-75-media-updates.md

```diff
@@ -0,0 +1,175 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the audio/video updates in Chrome 75: predicting whether playback will be smooth and power efficient for encrypted media and support of the video element's "playsInline" attribute hint.
+
+{# wf_updated_on: 2019-07-24 #}
+{# wf_published_on: 2019-07-22 #}
+{# wf_tags: news,chrome75,media #}
+{# wf_featured_image: /web/updates/images/generic/play-outline.png #}
+{# wf_featured_snippet: A round up of the audio/video updates in Chrome 75: predicting whether playback will be smooth and power efficient for encrypted media and support of the video element's "playsInline" attribute hint. #}
+{# wf_blink_components: Blink>Media #}
+
+# Audio/Video Updates in Chrome 75 {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Chrome's media capabilities were updated in version 75. In this article, I'll
+discuss those new features which include:
+
+- Predicting whether playback will be smooth and power efficient for encrypted
+  media.
+- Support of the video element's `playsInline` attribute hint.
+
+## Encrypted Media: Decoding Info API  {: #encrypted-media}
+
+Since Chrome 66, web developers have been able to use the [Decoding Info API] to
+query the browser about the clear content decoding abilities of the device based
+on information such as the codecs, profile, resolution, bitrates, etc. It
+indicates whether the playback will be smooth (timely) and power efficient based
+on previous playback statistics recorded by the browser.
+
+The [Media Capabilities API] specification has since been updated to handle
+encrypted media configurations as well so that websites using encrypted media
+(EME) can select the optimal media streams.
+
+In a nutshell, here’s how the Decoding Info API for EME works. Give a try to the
+[official sample].
+
+    const encryptedMediaConfig = {
+      type: 'media-source', // or 'file'
+      video: {
+        contentType: 'video/webm; codecs="vp09.00.10.08"',
+        width: 1920,
+        height: 1080,
+        bitrate: 2646242, // number of bits used to encode a second of video
+        framerate: '25' // number of frames used in one second
+      },
+      keySystemConfiguration: {
+        keySystem: 'com.widevine.alpha',
+        videoRobustness: 'SW_SECURE_DECODE' // Widevine L3
+      }
+    };
+
+    navigator.mediaCapabilities.decodingInfo(encryptedMediaConfig).then(result => {
```

Comment:
`MediaCapabilites.decodingInfo()` is shorter than the fully qualified call and it works. Is there a particular reason you've included `navigator`?

Reply:
`MediaCapabilities.decodingInfo()` is undefined. I'm surprised it works for you.

![image](https://user-images.githubusercontent.com/634478/61929759-2ac6fc00-af7c-11e9-9ceb-35df59d5ba21.png)

Reply:
`navigator.mediaCapabilities` is an instance of `MediaCapabilities`. To get the decoding info, you need to use the instance, which is exposed via the `navigator`.

Reply:
I can see it in the console. Is there something that didn't get finished?

Reply:
As discussed offline, `navigator.mediaCapabilities` works.